games.py

```diff
@@ -255,7 +258,9 @@ def actions(self, state):
 
     def result(self, state, move):
         if move not in state.moves:
-            return state  # Illegal move has no effect
+            return GameState(to_move=('O' if state.to_move == 'X' else 'X'),
```

Comment:
I'll accept this for now, but in general, this isn't right: the to_move need not be 'O' and 'X' in other games.